tests/unit/libbpf_test.cpp

```diff
@@ -17,6 +17,13 @@
 #include <stop_token>
 #include <thread>
 
+#if !defined(MAX_TAIL_CALL_CNT)
+#define MAX_TAIL_CALL_CNT 32
```

Comment:
Is there a reason why this can't be defined in one of the published 'ebpf-for-windows' headers and include that header here?

Reply:
It's already defined in `include\bpf_helpers.h`

Reply:
From an ebpf app writer's perspective though, this is still confusing.  I'd expect the inclusion of the 'one true header' to fix such internal compatibility issues.  All the more so for such key definitions.